conda/resolve.py

```diff
@@ -735,7 +735,8 @@ def remove(self, specs, installed):
     def solve(self, specs, returnall=False, _remove=False):
         # type: (List[str], bool) -> List[Dist]
         try:
```

Comment:
Changes here in `resolve.py` are just removing writes to `sys.stdout` when we shouldn't be writing there.